source/includes/_api_account_linking.md

```diff
@@ -219,8 +220,8 @@ print(data.decode("utf-8"))
 |Parameter	|Type	|Required	|Default	|Description	|
 |---	|---	|---	|---	|---	|
 |partner_user_id	|String(255)	|TRUE	|-	|ID generated by Partner for a specific user/customer	|
-|phone_number	|String(30)	|TRUE	|-	|The user phone number (format 08xxxxxxxxxx)	|
-|payment_method	|String(255)	|TRUE	|-	|The payment method type that user want to linked to (Currently only support `EWALLET`. In the future we will have another payment method type, such as `CARDS`)	|
+|phone_number	|String(20)	|TRUE	|-	|The user phone number (format 08xxxxxxxxxx)	|
+|payment_method	|String(255)	|TRUE	|-	|The payment method type that user want to linked to (currently only support `EWALLET`) |
```

Comment:
sorry penasaran, kenapa keterangan `In the future ...` dihapus ya?

Reply:
Oh karena dikit lagi aku release api docs ccdc sih. Tapi baru aware juga keknya gaperlu dihapus. Ku update ya